db/migrations/20221129064549_writings.sql

```diff
@@ -0,0 +1,17 @@
+-- migrate:up
+CREATE TABLE writings(
+id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+title VARCHAR(200) NOT NULL UNIQUE,
+content VARCHAR(5000) NOT NULL,
+header_image VARCHAR(2500) NOT NULL,
+price INT NOT NULL,
+category_id INT NOT NULL,
+user_id INT NOT NULL,
+created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+updated_at TIMESTAMP DEFAULT NULL ON UPDATE CURRENT_TIMESTAMP,
+CONSTRAINT category_id FOREIGN KEY (category_id) REFERENCES categories(id),
```

Comment:
```suggestion
CONSTRAINT wirtings_category_id_fkey FOREIGN KEY (category_id) REFERENCES categories(id),
```
constraint 명을 category_id로 하게 되면 다른 테이블과 중복이 될 수 있습니다.